### PR TITLE
Port flower mechanics from Industrial Apiary to Mega Apiary

### PR DESCRIPTION
**Pull request by kuba6000**

### CODE REVIEW

**Comment by Runakai1** [comment]
bad bad.

**Comment by boubou19** [comment]
this is really not needed.

**Comment by boubou19** [comment]
Also @kuba6000 given this is a nerf, you might want to describe what you actually changed so everyone can understand what's happening and discuss it before it's merged. No more shady nerfs please.

**Comment by kuba6000** [comment]
image
image
image
image

**Comment by BlueWeabo** [comment]
its not just those two flowers that give the flower pulp right? Also is there any way to produce said flowers without lag?

**Comment by Runakai1** [comment]
I'd like to speak against this nerf because adding more and more trivial lines to GTNH for the sake of another line is something we should stop doing and start doing creative stuff. 
Especiall a VERY simple line that is required in UV just because..

**Comment by boubou19** [comment]
my concerns:
- The line adds nothing much but bloat on its own. If you want to spam this, you'll have to spam 3 machines. Given how high this multi is getting tiered to the pack, it adds no real value, as it's neither hard nor intestesting to auto a 3 machine setup the regular way via piping.

- This nerf is only applied to the mega apiary, meaning that people spaming bees will avoid using mega apiary for the sake of tps by avoiding the machine spam

- Flowers aren't that easy to get: you can go iron golem farm to get poppies, but that is gated to UEV+ because of the DE spawners are the only type of spawner that can accept the iron golem, powered spawners are blacklisting it. You then need to get the iron golem soul, so it's quite grindy. You can also get flowers by using flip flop piston clock to move the dir blocks so flowers spawned by bees drop on the ground and you get them with something like and ender collector, with WAs on normal apiaries to speed up flower spawn, which is silly because we don't want to encourage that at all. Or you manually grind them with bone meal on grass. __In the end, it's only stupid ways of getting flowers, generating tons of lag.__

- This has not been discussed with the team in our dev channels, so just for that, the PR should be blocked.

In short, i am against this nerf.

**Comment by mitchej123** [comment]
The multi was added, in my mind, to move away from the WA + industrial apiary spam + lag.  Adding another line, with more machines seems counterproductive.

**Comment by Dream-Master** [comment]
We have many unused Binnie machines and fluids that would fit. Maybe we make binnie stuff better und use them inside gt instead of using so much gt machines

**Comment by BlueWeabo** [comment]
> We have many unused Binnie machines and fluids that would fit. Maybe we make binnie stuff better und use them inside gt instead of using so much gt machines

which fluids are you thinking of using for this? DNA is something that requires a constant drone production to produce and you get very little of per drone. It would require the player to spam single block apiaries making the MApiary useless as its trying to prevent that.

**Comment by kuba6000** [comment]
image
image

**Comment by Runakai1** [comment]
This isn't a required change, still. Means of getting flowers is tedious and laggy.
Place same flowers inside the mapiary and call it a day.

**Comment by Runakai1** [comment]
> We have many unused Binnie machines and fluids that would fit. Maybe we make binnie stuff better und use them inside gt instead of using so much gt machines

Binnie machines arent meant for what kuba has in mind.

**Comment by Runakai1** [comment]
Place same flowers inside the mapiary and call it a day.

**Review by Alastors — CHANGES_REQUESTED** [review]
These aren't needed, seriously

**Comment by Alastors** [comment]
Can we get the math behind why this is needed and will be beneficial?

**Comment by Dream-Master** [comment]
> > We have many unused Binnie machines and fluids that would fit. Maybe we make binnie stuff better und use them inside gt instead of using so much gt machines
> 
> which fluids are you thinking of using for this? DNA is something that requires a constant drone production to produce and you get very little of per drone. It would require the player to spam single block apiaries making the MApiary useless as its trying to prevent that.

i mean more we can use parts from Binni after we change it . Binni is much underpowered rn but i guess if we change that it can be more usefull

**Comment by Runakai1** [comment]
> > > We have many unused Binnie machines and fluids that would fit. Maybe we make binnie stuff better und use them inside gt instead of using so much gt machines
> > 
> > 
> > which fluids are you thinking of using for this? DNA is something that requires a constant drone production to produce and you get very little of per drone. It would require the player to spam single block apiaries making the MApiary useless as its trying to prevent that.
> 
> i mean more we can use parts from Binni after we change it . Binni is much underpowered rn but i guess if we change that it can be more usefull

First the thinking, then the change

**Comment by Runakai1** [comment]
@kuba6000 the other concerns have not been addressed by you except that you now use 2 machines less. Which isn't doing anything favourable for this PR. 
Can you address them? Otherwise I don't see the point of not closing this PR.

**Comment by Dream-Master** [comment]
Ok here is the plan i discussed with Kuba and Runakai so far:
We add some kind of fluid to the bee production process like the flower water or a sugar water(use EIG products (flowers, sugar etc))
And we add bee products (fluid) to Eig growth stuff and remove water(we create a new Wheed ex fluid use napthenic acid, pollen and maybe more ingrediensts)
both Multies should be in one tier (MApiary and EIG)

**Comment by Runakai1** [comment]
> Ok here is the plan i discussed with Kuba and Runakai so far: We add some kind of fluid to the bee production process like the flower water or a sugar water(use EIG products (flowers, sugar etc)) And we add bee products (fluid) to Eig growth stuff and remove water(we create a new Wheed ex fluid use napthenic acid, pollen and maybe more ingrediensts) both Multies should be in one tier (MApiary and EIG)

I agree 🙏

**Comment by kuba6000** [comment]
image
image
image

**Comment by LewisSaber** [comment]
what if bee want nether wart( i know you can overwrite with gendustry but still)

**Comment by kuba6000** [comment]
> what if bee want nether wart( i know you can overwrite with gendustry but still)

Hm, I can make it check two blocks tall then, so you can place a block and a flower on it or allow placing other blocks than dirt as a floor

**Comment by LewisSaber** [comment]
there is also cactus, thats only one that need 2 blocks i can recall

**Review by Runakai1 — APPROVED** [review]

**Comment by Runakai1** [comment]
Bees.
